### PR TITLE
Updated the App Services documentation

### PR DESCRIPTION
**Pull request by NicoleYarroch**
Added 2 subsections to **Creating an App Service**:
* Handling App Service Subscribers
* Sending Files to a Subscriber

This pull request adds new content.

## Summary of Changes
Adds more data to the App Service documentation.

### CODE REVIEW

File: docs/Other SDL Features/Creating an App Service/index.md [hunk 1]
```diff
@@ -203,4 +207,109 @@ NotificationCenter.default.addObserver(self, selector: #selector(performAppServi
 
     sdlManager.sendRPC(response)
 }
+```
+
+## Handling App Service Subscribers
```
Comment (joeljfischer) [inline docs/Other SDL Features/Creating an App Service/index.md]: None of this section is needed for a service provider. It's all handled by the system.

Comment (NicoleYarroch) [reply docs/Other SDL Features/Creating an App Service/index.md]: I just confirmed with Jake that the module does not handle `GetAppServiceData ` requests, the producer does. They were talking about having the module handle the request but they decided to wait to add it in a future proposal.

Comment (joeljfischer) [reply docs/Other SDL Features/Creating an App Service/index.md]: 🍳 🤦‍♂️

**Comment by joeljfischer** [comment]
Moved much of this content into PR #135

**Pull request closed without merging**